test_ngtf.py

```diff
@@ -260,11 +268,26 @@ def main():
     build_dir = 'build'
     venv_dir = 'build/venv-tf-py3'
 
-    # Run the bazel based buil
-    run_bazel_build_test(venv_dir, build_dir)
+    if (platform.system() != 'Darwin'):
+        # Run the bazel based buil
+        run_bazel_build_test(venv_dir, build_dir)
 
     # First run the C++ gtests
-    run_ngtf_gtests(build_dir)
+    run_ngtf_gtests(build_dir,None)
+
+    # If the GPU tests are requested, then run them as well
+    if (arguments.gpu_unit_tests_enable):
+        os.environ['NGRAPH_TF_BACKEND'] = 'GPU'
+        run_ngtf_gtests(
+            build_dir, 
+            '-ArrayOps.Quanti*:ArrayOps.Dequant*:BackendManager.BackendAssignment:'
+            'MathOps.AnyKeepDims:MathOps.AnyNegativeAxis:MathOps.AnyPositiveAxis:'
+            'MathOps.AllKeepDims:MathOps.AllNegativeAxis:MathOps.AllPositiveAxis:'
+            'NNOps.Qu*:NNOps.SoftmaxZeroDimTest*:'
+            'NNOps.SparseSoftmaxCrossEntropyWithLogits'
+        )
```

Comment:
I suppose these tests fail on GPU?

Reply:
For now we have added special flags for GPU (`gpu_unit_tests_enable` and `ngraph_also_build_gpu_backend`). When running this test suite on other backends, do we need to do anything special, in which case we might want to refactor this later to have a single unified flag for all backends


eg: `--ngraph_build_other_backend=GPU,INTERPRETER` and `--unit_test_backend=GPU,INTERPRETER`.

Each backend might have its exclusion list of tests etc.